docs/src/modules/java-protobuf/partials/actions.adoc

```diff
@@ -4,7 +4,9 @@ Actions are stateless functions that can be used to implement different uses cas
 
 - a pure function
 - request conversion - you can use Actions to convert incoming data into a different format before forwarding a call to a different component.
-- publish and subscribe to events 
+- publish messages to a topic
+- subscribe to events from an event-sourced entity
```

Comment:
```suggestion
- subscribe to events from an Event Sourced Entity.
```
I see this "Event Sourced Entity" is the standard way of writing it
Also I can see most the bullet points in other pages ending with a period.